src/etc/inc/certs.inc

```diff
@@ -622,6 +647,40 @@ function cert_get_publickey($str_crt, $decode = true, $type = "crt") {
 			break;
 	}
 	unlink($certfn);
+	return ($out !== false ? implode("\n", $out) : false);
```

Comment:
The comparison here should be inside parenthesis, so `return (($out !== false) ? implode("\n", $out) : false);` for example

Reply:
ACK